app/models/video_stream_moderator.py

```diff
@@ -0,0 +1,21 @@
+from app.models import db
+from app.models.base import SoftDeletionModel
+
+
+class VideoStreamModerator(SoftDeletionModel):
+    __tablename__ = 'video_stream_moderator'
+
+    id = db.Column(db.Integer, primary_key=True)
+
+    event_id = db.Column(
```

Comment:
???

Reply:
I was just going to remove that. in next commit. just wanted to know what's next. Means if I am understanding the feature correctly or not.

Reply:
There is not video_stream_id here.

As discussed in the meeting, if you see meeting notes, it is written there (video_stream_id, user_id)

Now you'll have to remove all the API and schema and whatnot

That's why it was throwing error when adding relationship in the model